src/Home.tsx

```diff
@@ -11,8 +11,8 @@ import {
 } from 'react-native';
 import styles from './styles';
 
-import DaySelector from './components/DaySelector';
-import Meals from './components/Meals';
+import DaySelector from './components/daySelector/DaySelector';
```

Comment:
Can we please structure the folders/files differently? 
DaySelector/
  DaySelector.tsx
  DaySelector.spec.ts
  index.ts

Reply:
what is the index.ts for?

Reply:
you mean the index.ts should be in the root of the component folder?

Reply:
also, and also in each folder of the components